src/TopStats.re

```diff
@@ -29,7 +30,18 @@ let byTupleValue = ((_, r1), (_, r2)) =>
   };
 
 [@react.component]
-let make = (~title, ~resultsWithMap: temp_resultsWithRatingMap, ~startDate) =>
+let make =
+    (
+      ~communityName,
+      ~title,
+      ~resultsWithMap: temp_resultsWithRatingMap,
+      ~startDate,
+    ) => {
+  let settingsQueryConfig =
```

Comment:
You could make a custom hook, like `useCommunitySettings` that would return the same variant but with data  already converted with `toCommunitySettings`. It is going to be awesome!

Reply:
Done. Awesome indeed - thanks! 😄